### ISSUE TITLE
Keep Attribute Editor properties expanded

### ISSUE THREAD

**Issue by antares1470** [constellation-app/constellation#1128]
### Prerequisites

- [x] Put an X between the brackets on this line if you have done all of
    the following:

    -   Running the latest version of Constellation

    -   Attached the ***Support Package*** via `Help` &gt;
        `Support Package`

    -   Checked the FAQs:
        https://github.com/constellation-app/constellation/wiki/FAQ

    -   Checked that your issue isn’t already filed:
        https://github.com/constellation-app/constellation/issues

    -   Checked that there is not already a module that provides the
        described functionality:
        https://github.com/constellation-app/constellation/wiki/Catalogue-of-Repositories

### Description

Whenever the graph selection changes, any expanded node or transaction attributes contract to display "<Multiple Values>", rather than remain expanded and populate newly selected values. A toggle next to each attribute allowing users to keep the values expanded (or expand it for them when the selection changes) would save a lot of time doing it manually.

### Steps to Reproduce

1. Open a graph and the Attribute Editor
2. Select a set of nodes or transactions such that an attribute has more than one value for the selection
3. Expand an attribute with multiple values
4. Change the graph selection such that the chosen attribute still has multiple values

**Expected behaviour:** There is an option to keep the attribute values expanded

**Actual behaviour:** Attribute values always contract

**Reproduces how often:** 100%

**Comment by Auriga2** [comment]
I added an option next to the Expand/collapse button, with a too tip text as below: 
1.  Any suggestion to improve it's appearance and location?
2. Should this option work across all graphs? Current version only works within a graph.
3. When zoom in/out currently it only works for the attributes this new option is checked. I guess this should be for everything. (Currently working on it- it's triggering attibuteValueHasChanged = true for at least one attribute)
![keep attributes expanded](https://user-images.githubusercontent.com/64826928/126583614-37dd3af7-6c07-47a1-9f14-4f3acc2e1f07.gif)

**Comment by GammaVel** [comment]
Thanks @Auriga2 for making these improvements. I noticed that the checkbox didn't get implemented as initially mentioned in the issue description. As you found, if you change graphs the attributes that are expanded don't retain state when you go back to the original graph. 

I also noticed if you are looking at multiple values in the transaction section and then select multiple transactions and nodes, Constellation will auto scroll back up to the Nodes section, not the transaction section ( I believe this is because of the priority of the nodes.   Cheers :-)
